lib/cmds/ocr_cmds/get-suggestions.js

```diff
@@ -2,8 +2,28 @@
 
 const querystring = require('querystring');
 const moment = require('moment');
+const _find = require('lodash/find');
 const { get } = require('../../api');
+const { get: fhirGet, getAccount } = require('../../fhir');
 const print = require('../../print');
+const sleep = require('../../sleep');
+
+const getStatus = async argv => {
+  let status = 'RUNNING';
+  do {
+    const account = getAccount(argv);
+    const url = `/${account}/dstu3/DocumentReference/${argv.documentId}`;
+    const response = await fhirGet(argv, url);
+    const tag = response.data && response.data.meta && response.data.meta.tag;
+    if (!tag) {
+      status = 'ERROR';
+      break;
+    }
+    status = _find(tag, {system: 'http://lifeomic.com/ocr/document/status'}).code;
+    if (status.includes('RUNNING')) await sleep(1000);
+  } while (status.includes('RUNNING'));
```

Comment:
I'm concerned you could get stuck in an infinite loop if the document doesn't finish (e.g. a step throws an error but doesn't update the status to `error`). I'm also wary of constantly retrying with extremely large documents that take a long time to process. I'd suggest a maxAttempts that is checked in your `while` condition